lib/dor/text_extraction/abbyy/results.rb

```diff
@@ -64,6 +64,9 @@ def move_result_files(destination_dir)
             dest_path = File.join(destination_dir, File.basename(path))
             @logger.info("moving #{path} to #{dest_path}")
             FileUtils.mv(path, dest_path)
+            # ensure that the file can be read by anyone so that preservation_robots can read it
+            # during preservationIngestWF transfer-object which runs as user pres
+            FileUtils.chmod('u=wr,g=wr,o=r', dest_path)
           end
```

Comment:
possibly remove if we can have abbyy/ops set these correctly

Reply:
Yeah, I don't know if that's possible. Perhaps it could be ticketed as follow up work?